### PR TITLE
Documentation: New extended tutorial for Liqo.

### PR DESCRIPTION
**Pull request by Andreagit97**
# Description

This pull request provides a more in-depth tutorial showing how to use Liqo's main features.
The tutorial is inserted in the /docs/pages/gettingstarted section with the already present "hello world" tutorial.

### CODE REVIEW

File: docs/pages/GettingStarted/Extended/configure_topologies.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+title: Configure yours deployment topologies
+weight: 3
+---
+
+In this tutorial we want to simulate 3 realistic scenarios, considering the cluster-1 as the **cluster home**:
```
Comment (frisso) [inline docs/pages/GettingStarted/Extended/configure_topologies.md:6]: I would suggest to describe the above 3 scenarios in a more high-level way. The current description is fine, but it's a technical description, not business. We should present them both.
E.g., (for the first case): This simulates a case in which a service has to replicated on multiple clusters, e.g., all the ones that belongs to the same geographical area.

File: docs/pages/GettingStarted/Extended/kind.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+kube-system          kube-apiserver-cluster1-control-plane            1/1     Running   0          57m
+kube-system          kube-controller-manager-cluster1-control-plane   1/1     Running   0          57m
+kube-system          kube-proxy-88m2g                                 1/1     Running   0          57m
+kube-system          kube-proxy-zctxs                                 1/1     Running   0          57m
+kube-system          kube-scheduler-cluster1-control-plane            1/1     Running   0          57m
+local-path-storage   local-path-provisioner-78776bfc44-rk58g          1/1     Running   0          57m
+```
+
+If the above command return an output similar to this for each cluster, you are ready to move 
+forward to the [next step](../install): the installation of Liqo..
```
Comment (frisso) [inline docs/pages/GettingStarted/Extended/kind.md:83]:
```suggestion
forward to the [next step](../install): the installation of Liqo.
```

File: docs/pages/GettingStarted/Extended/configure_topologies.md (hunk 1 above)
Comment (Andreagit97) [reply docs/pages/GettingStarted/Extended/configure_topologies.md:6]: yes thanks for the advice, we are just working to improve this aspect.

File: docs/pages/GettingStarted/Extended/kind.md (hunk 2 above)
Comment (Andreagit97) [reply docs/pages/GettingStarted/Extended/kind.md:83]: ty

File: docs/pages/gettingstarted/_index.md [hunk 3]
```diff
@@ -3,13 +3,24 @@ title: Quickstart
 weight: 1
 ---
 
-## The fast path to Liqo
+### Introduction
 
-The following steps will guide you through a tour to learn how to install Liqo and experiment with its most notable capabilities.
+Liqo enables the possibility to create simple and straightforward multi-cluster topologies, proposing a seamless multi-cluster model which dynamically allows the pods and services offloading to remote clusters.
```
Comment (frisso) [inline docs/pages/gettingstarted/_index.md:8]:
```suggestion
Liqo enables the creation of arbitrary multi-cluster topologies, proposing a seamless multi-cluster model which dynamically allows the pods and services to be offloaded on remote clusters.
```

**Review by palexster — APPROVED** [review]

**Review by alacuku — APPROVED** [review]

**Pull request merged**